src/test/logical_query_plan/union_node_test.cpp

```diff
@@ -91,12 +91,6 @@ TEST_F(UnionNodeTest, DiscardFunctionalDependencies) {
   auto validate_node = ValidateNode::make(stored_table_node);
   EXPECT_EQ(validate_node->functional_dependencies().size(), 1);
 
-  // Test UnionAll (discard FDs)
-  auto union_all_node = UnionNode::make(SetOperationMode::All);
-  union_all_node->set_left_input(stored_table_node);
-  union_all_node->set_right_input(stored_table_node);
-  EXPECT_EQ(union_all_node->functional_dependencies().size(), 0);
-
```

Comment:
Just curious: Why not have such a test?

Reply:
Because I replaced the UnionAll handling with an assertion. We haven't decided what expressions are returned if we run a union with multiple input tables. Writing tests about this pretends that we do.

Reply:
Alright, makes sense.